integreat_cms/cms/linklists.py

```diff
@@ -38,18 +59,20 @@ class PageTranslationLinklist(Linklist):
     @classmethod
     def filter_callable(cls, objects):
         """
-        Get only latest versions for non-archived pages
+        Get only latest versions for non-archived pages in acitive languages
 
         :param objects: Objects to be filtered
         :type objects: ~django.db.models.query.QuerySet
 
         :return: Objects that passed the filter
         :rtype: ~django.db.models.query.QuerySet
         """
-        # Since filtering the archived pages causes a lot of overhead, we only want this during the findlinks management command.
-        if settings.LINKCHECK_EXCLUDE_ARCHIVED_PAGES:
+        # Because of large overhead, we only want this filter during the findlinks management command.
```

Comment:
```suggestion
        # Apply filter of parent class
        objects = super().filter_callable(objects)
        # Because of large overhead, we only want this filter during the findlinks management command.
```
Overriding a method will replace it completely, so the code defined in the parent class is not run in this case.
So we need to explicitly call `super()` if we want to extend the base class (and not replace it).
This is a pretty fundamental concept in Python, so keep this in mind.